docs/whats-new/release-candidate-1.md

```diff
@@ -0,0 +1,358 @@
+
+.NET Aspire 9.0 introduces a new project template for creating .NET Aspire projects. This template is available via the `dotnet new` command. These can be installed by running the following command:
+
+```shell
+dotnet new install Aspire.ProjectTemplates::9.0.0-rc.1.24511.1
+```
+
+![dotnet new templates](images/templates.png)
+
+## Dashboard
+
+### Stop/Start/Restart Resources
+
+The most requested feature for the dashboard is the ability to stop, start, and restart resources. This feature is now available in .NET Aspire 9.
+
+<video controls src="videos/start-stop-restart.mp4" title="Title"></video>
+
+This feature works for projects, containers and executables. It enables restarting indidivual resources without having to restart the entire app host. For projects, if the debugger is attached, on restart, the debugger will be re-attached.
+
+- Responsive design
+- Mobile friendly
+
+## App Host (Orchestration)
+
+### Waiting for dependencies
+
+The app host now supports waiting for dependencies to be ready before starting another resource. This is useful when a resource sources take a long time to start up. The app host will wait for the resource to be ready before starting the dependent resource.
+
+```C#
+var builder = DistributedApplication.CreateBuilder(args);
+
+var queue = builder.AddRabbitMQ("rabbit");
+
+builder.AddProject<Projects.WebApplication1>("api")
+    .WithReference(queue).WaitFor(queue);
+
+builder.Build().Run();
+```
+
+When the app host starts, it will wait for the `rabbit` resource to be ready before starting the `api` resource:
+
+<video controls src="videos/waitfor.mp4" title="Title"></video>
+
+There are 2 methods exposed to wait for a resource:
+
+1. `WaitFor` - Wait for a resource to be ready before starting another resource.
+2. `WaitForCompletion` - Wait for a resource to complete before starting another resource.
+
+#### Resource Health Checks
+
+`WaitFor` uses health checks to determine if a resource is ready. If a resource does not have any health checks, the app host will wait for the resource to be in the "Running" state before starting the dependent resource.
+
+You can now attach health checks to resources. These health checks use the `IHealthCheck` interface from the `Microsoft.Extensions.Diagnostics.HealthChecks` namespace.
+
+```C#
+var builder = DistributedApplication.CreateBuilder(args);
+
+var catalogApi = builder.AddContainer("catalog-api", "catalog-api")
+                 .WithHttpEndpoint(targetPort: 8080)
+                 .WithHttpHealthCheck("/healthz");
+
+builder.AddProject<Projects.WebApplication1>("store")
+    .WithReference(catalogApi.GetEndpoint("http"))
+    .WaitFor(catalogApi);
+
+builder.Build().Run();
+```
+
+The above example adds a health check to the `catalog-api` resource. The app host will wait for the health check to return a healthy status before starting the `store` resource. It'll determine that the resource is ready when the the /healthz endpoint returns a 200 status code.
```

Comment:
A screenshot showing how we surface health check data in the dashboard would be good here.

Reply:
I added some screenshots. @mitchdenny they're in a section I think you wrote, which I had to wrangle a little. PTAL.